File: src/main/java/Entities/Text.java
package Entities;

public class Text extends Entity {

	private static final String DIV = "div";
	private static final String STATUS = "status";
	private static final String TEXT = "Text";

	public Text(String status, String div) {
		super(TEXT, null, null);
		entity.put(STATUS, status);
		entity.put(DIV, div);
	}
}

File: src/main/webapp/js/Integration/Text.js
function Text(status, div) {
	this.status = status;
	this.div = div;
};

Text.prototype.prettyPrint = function() {
	return "Status: " + this.status
	+ " Div: " + this.div;
};

Text.prototype.jsonPrint = function() {
	return {	status: this.status,
		div: this.div
	};
};

module.exports = Text;
